freyja/tests/test_utils.py

```diff
@@ -29,6 +29,153 @@ def test_prep(self):
+            },
+            {
+                'Lineages': {
+                    'AY.48': {
+                        'name': 'A',
+                        'members': ['B.1.617*', 'B.1.252'],
+                        'color': 'orange'
+                        },
+                    'Q.3': {
+                        'name': 'B',
+                        'members': ['Q.3'],
+                        'color': 'green'
+                        }
+                    }
+            },
+        {
+            'VOC': {
+                'Delta': {
+                    'name':'Delta',
+                    'color': 'default'
+                    }
+                }
+            },
+        ]
+
+        output_valid_configs = [
+            {
+                'Lineages': {
+                    'AY.48': {
+                        'name': 'A',
+                        'members': ['B.1.617*', 'B.1.252'],
+                        'color': 'orange'
+                        },
+                    'Q.3': {
+                        'name': 'B',
+                        'members': ['Q.3'],
+                        'color': 'green'
+                        }
+                    }, 
+                'VOC': {
+                    'Delta': {
+                        'name':'Delta',
+                        'color': 'default'
+                        }
+                    }
+            },
+            {
+                'Lineages': {
+                    'AY.48': {
+                        'name': 'A',
+                        'members': ['B.1.617*', 'B.1.252'],
+                        'color': 'orange'
+                        },
+                    'Q.3': {
+                        'name': 'B',
+                        'members': ['Q.3'],
+                        'color': 'green'
+                        }
+                    }, 
+                'VOC': {}
+            },
+            {
+                'Lineages': {}, 
+                'VOC': {
+                    'Delta': {
+                        'name':'Delta',
+                        'color': 'default'
+                        }
+                    }
+            },
+        ]
+
+
+        # config dictionaries whose format is invalid.
+        invalid_configs = [
+            {
+                'Lineages': {
+                    'AY.48': {
+                        'name': 'A',
+                        'members': ['B.1.617*', 'B.1.252'],
+                        },
+                    'Q.3': {
+                        'name': 'B',
+                        'members': ['Q.3'],
+                        'color': 'green'
+                        }
+                    }, 
+                'VOC': {
+                    'Delta': {
+                        'name':'Delta',
+                        'color': 'default'
+                        }
+                    }
+            },
+            {
+                'Lineages': {
+                    'AY.48': {
+                        'name': 'A',
```

Comment:
This is ok since no one will be interacting with it, but a bit confusing for the reasons mentioned in the other comment.

Reply:
Fixed